build.gradle

```diff
@@ -54,9 +57,14 @@ afterEvaluate {
     }
 }
 
+
 allprojects {
-    group = 'org.spekframework.spek2'
+    // This causes a circular dependency for some reason.
```

Comment:
// Ah, lots of plugins rely on `groupId` property, I've faced similar issue before